site/website/blog/2020-10-05-jet-43-is-released.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Jet 4.3 is Released
+author: Marko Topolnik
+authorURL: https://twitter.com/mtopolnik
+authorImageURL: https://pbs.twimg.com/profile_images/922726943614783488/Pb5DDGWF_400x400.jpg
+---
+
+Today we're releasing Hazelcast Jet 4.3, our fourth release in 2020!
```

Comment:
Considering all the comments I've received in the past about not writing bland enough documentation, I have to comment: the second part is pure bragging, no information content whatsoever. Don't get me wrong, I like it, but from what I understood, that's a bad sign!

Reply:
But this is a blog post, not docs :)